### DISCUSSION TITLE
Add docs (or potentially a recipe) for proxying requests from an endpoint

### DISCUSSION THREAD

**Discussion by bholmesdev** [withastro/docs#12191] [Ideas]
### 📚 Subject area/topic

SSR

### 📋 Suggested page

https://docs.astro.build/en/guides/endpoints/#server-endpoints-api-routes

### 📋 General description or bullet points (if proposing new content)

[We received a roadmap discussion for adding better proxy support from API endpoints](https://github.com/withastro/roadmap/discussions/665). Based on the replies, this is possible today by returning the result of a `fetch()` call. This capability is not documented, nor is it very intuitive. Would be worth an explicit guide on Proxying Requests.

**Comment by florian-lefebvre** [comment]
Worth noting vite has an api for it too https://vite.dev/config/server-options#server-proxy

**Comment by sarah11918** [comment]
This does sound like a great recipe that we can link to using our `<RelatedRecipe/>` component from the Endpoints page!

Putting a `help wanted` on this one in case anyone would like to put something together!

**Comment by sarah11918** [comment]
pinging for freshness! We would still happily accept some content about this!

**Comment by florian-lefebvre** [comment]
I'll note there seems to be another way for manual usage, see the source of Hono proxy helper: https://github.com/honojs/hono/blob/main/src/helper/proxy/index.ts
